README.md

```
@@ -32,8 +32,27 @@ Command line without opening a browser (headless mode)
 yarn cypress run --config excludeSpecPattern=cypress/e2e/filesearch/Filesearch.spec.ts
 ```
 
 This will skip running the slow filesearch spec when executing your tests locally.
+
+## Flow smoke test
+
+The flow smoke test (`cypress/e2e/smoke.spec.ts`) runs against a live Glific instance, validates the `smoke-test` flow via the simulator, and reports pass/fail to Instatus.
+
+Run locally (set prod credentials in `.env` or export `CYPRESS_*` vars; regular E2E tests keep using `phone` / `password` in `cypress.config.ts` for glific.test):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)